### DISCUSSION TITLE
Vote for xxchan as RisingLight maintainer

### DISCUSSION THREAD

**Discussion by skyzh** [risinglightdb/risinglight#628] [Member]
Since @xxchan has been voted to RisingLight member, he has been continuously working on various parts of the RisingLight project. Here's a list of PR and issue he'd worked on:

https://github.com/risinglightdb/risinglight/pulls?q=author%3Axxchan
https://github.com/risinglightdb/risinglight/issues/created_by/xxchan

At the same time, @xxchan also reviews PRs from the community, and gives valuable feedbacks for us. He is now an indispensable part of the RisingLight project, who is able to decide the overall direction for the project. Therefore, I vote him as RisingLight maintainer.

A maintainer vote requires:

* At least 2 votes from RisingLight maintainers.
* At least 48 hours after the proposal.
* No vetos within the vote process.

Welcome to vote with 👍 and 👎!

**Comment by skyzh** [comment]
With 3 votes from @wangrunji0408 @st1page and @skyzh, we'd like to welcome @xxchan as RisingLight maintainer!

**Reply by xxchan** [reply]
🫡 It is a great honor for me 🥰
